src/stream_wrap.cc

```diff
@@ -204,7 +198,7 @@ void StreamWrap::OnReadImpl(ssize_t nread,
     return;
   }
 
-  char* base = static_cast<char*>(node::Realloc(buf->base, nread));
+  char* base = node::UncheckedRealloc(buf->base, nread);
   CHECK_LE(static_cast<size_t>(nread), buf->len);
```

Comment:
I’ve moved the existing check and turned the `Realloc` into a checked one, I guess that makes sense.